doc/content/getting-started/troubleshooting/_index.md

```diff
@@ -31,6 +35,10 @@ Your gateway receives traffic from all devices in range, and {{% tts %}} drops t
 
 In rare cases, this may also happen to ABP devices if a device resets its frame counters (usually because of a power cycle) and can not be found by the session. In this case, it is necessary to re-activate the device. OTAA prevents this problem from occurring, and is more secure in general, so it is the preferred form of activation when possible.
 
+### EUI Already Taken
+
+This error occurs when you try to add gateway or device with an EUI that already exists in your deployment. To find the duplicate, search from an account with Admin priveleges.
```

Comment:
This is a bit suggestive. E.g. a PCN user will usually not have the possibility to search for this with admin privileges.

Reply:
@htdvisser what's your suggestion for solution to EUI already taken

Reply:
In most cases, the API will return either one of:

- gateway_eui_taken (a gateway with EUI `{gateway_eui}` is already registered as `{gateway_id}`)
- end_device_euis_taken (an end device with JoinEUI `{join_eui}` and DevEUI `{dev_eui}` is already registered as `{device_id}` in application `{application_id}`)

The bare _eui_taken (EUI already taken)_ error is only shown in a multi-tenant deployment where the EUI is registered in a different tenant. In that case neither the user, nor an admin can search for the entity, because it doesn't exist in their tenant.